src/styles/Components/Card.sass

```diff
@@ -0,0 +1,53 @@
+.card
+  width: 40%
+  height: 50%
+  margin: 10px
+  box-shadow: 0 0 11px #aaa
+
+  &.-long
+    height: 100%
+
+  &__image
+    width: 100%
+    height: calc(100% - 150px)
+    background-image: url('../../../src/assets/images/airplane.jpg')
+    background-size: cover
+    background-repeat: no-repeat 
+ 
+  
+  &__body
+    width: 100%
+    height: 150px
```

Comment:
Rather than having a fixed value, it should adapt to the content
![Screenshot 2022-09-21 at 08 28 36](https://user-images.githubusercontent.com/2543423/191430470-c6b66d84-a8ee-4722-acd7-06d5a0858384.png)